### PR TITLE
Refactor FXIOS-5427 [v110] Remove SnapKit from TodayViewController

### PR DESCRIPTION
**Pull request by thatswinnie**
[Jira](https://mozilla-hub.atlassian.net/browse/FXIOS-5427)
#12682

### CODE REVIEW

File: Extensions/Today/TodayViewController.swift [hunk 1]
```diff
@@ -32,26 +31,30 @@ class TodayViewController: UIViewController, NCWidgetProviding, TodayWidgetAppea
     fileprivate lazy var newTabButton: ImageButtonWithLabel = {
```
Comment (yoanarios) [inline Extensions/Today/TodayViewController.swift:31]: nit: Clean up the `fileprivate` for `private`

File: Extensions/Today/TodayViewController.swift [hunk 2]
```diff
@@ -160,4 +161,27 @@ class TodayViewController: UIViewController, NCWidgetProviding, TodayWidgetAppea
...
+        widgetView.addSubview(buttonStackView)
+
+        NSLayoutConstraint.activate([
+            effectView.topAnchor.constraint(equalTo: view.topAnchor),
+            effectView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            effectView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            effectView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+
+            buttonStackView.topAnchor.constraint(equalTo: widgetView.topAnchor),
+            buttonStackView.leadingAnchor.constraint(equalTo: widgetView.leadingAnchor, constant: 5),
```
Comment (yoanarios) [inline Extensions/Today/TodayViewController.swift:182]: small nit: We could create an UX struct and add the constant values

**Comment by thatswinnie** [comment]
Not running PR again after the small changes. Last build was green:
Screenshot 2022-12-13 at 11 12 03

**Review by yoanarios — COMMENTED** [review]
Approved ✅

**Pull request merged**